src/api/decode.js

```diff
@@ -1,244 +1,340 @@
-/**
- * @brief Mobile Application logger - server side
- *
- *
- * @copyright Ooma Inc, 2015
- * @author: Dmitry Kudryavtsev
- */
-var event_type_dict ={
-    0:'SYSTEM',
-    1:'UI',
-    2:'SIP',
-    3:'RUNTIME',
-    4:'HM',
-    10:'UNDEFINED'
+var eventTypeDict = {
```

Comment:
there is an api to get and set these in the master branch on git. After that i think its just a mongo lookup?